decidim-admin/app/assets/javascripts/decidim/admin/application.js.es6

```diff
@@ -35,6 +35,17 @@ const pageLoad = () => {
       );
     }
   })
+
+  const $participatoryProcessScopeEnabled = $('#participatory_process_scope_enabled');
+  const $participatoryProcessScopeId = $("#participatory_process_scope_id");
+
+  if ($('.edit_participatory_process').length > 0) {
+    $participatoryProcessScopeEnabled.on('change', (event) => {
+      const checked = event.target.checked;
+      $participatoryProcessScopeId.attr("disabled", !checked);
+    })
+    $participatoryProcessScopeId.attr("disabled", !$participatoryProcessScopeEnabled.prop('checked'));
+  }
```

Comment:
Sure, no problem! I really hope someday we can use React components for this kind of things 😓 .